library/testing/TestSDKHelpers.h

```diff
@@ -1,61 +1,65 @@
 #ifndef TestSDKHelpers_h
 #define TestSDKHelpers_h
 
+#include <image.h>
+#include <window.h>
+
 #include <fstream>
 #include <iostream>
 
-class TestSDKHelpers
+namespace TestSDKHelpers
```

Comment:
I think you wanted to rename this file ?

Reply:
I thought it'd be nice if only the files containing actual tests were called `Test*` but I can't think of a better name for this one... it was probably not a very useful point to begin with anyway so unless it resonate with you we can just forget about it

Reply:
fine by me :)